public/app/features/explore/utils/links.test.ts

```diff
@@ -9,10 +9,22 @@ import {
   LinkModel,
   TimeRange,
 } from '@grafana/data';
+import { setTemplateSrv } from '@grafana/runtime';
 import { setLinkSrv } from '../../panel/panellinks/link_srv';
 import { setContextSrv } from '../../../core/services/context_srv';
 
 describe('getFieldLinksForExplore', () => {
+  beforeEach(() => {
+    setTemplateSrv({
+      replace(target, scopedVars, format) {
+        return target ?? '';
+      },
+      getVariables() {
+        return [];
+      },
+    });
+  });
+
```

Comment:
Because of circular dependencies this test could previously run without using `setTemplateSrv`. This isn't the case any longer.